lib/ledger_sync/adaptors/quickbooks_online/deposit/operations/find.rb

```diff
@@ -0,0 +1,36 @@
+module LedgerSync
+  module Adaptors
+    module QuickBooksOnline
+      module Deposit
+        module Operations
+          class Find < Operation::Find
+            class Contract < LedgerSync::Adaptors::Contract
+              schema do
+                required(:ledger_id).filled(:string)
+                required(:to_account).hash(Types::Reference)
+                required(:currency).maybe(:string)
+                required(:memo).maybe(:string)
+                required(:transaction_date).filled(:date?)
+                required(:exchange_rate).maybe(:float)
+                required(:line_items).array(Types::Reference)
```

Comment:
@jozefvaclavik - it looks like there's a `TotalAmt` field in the API that is read only. Can we maybe fetch that and expose it as an attribute you can read? There's no way to write to it, so we wouldn't let you set it in the create operation

Reply:
@mattgmarcus I'm not sure it's that easy.  The best way to get that info is  `result.operation.response` if it is not an attribute of the resource.  In my serializer PR, I am making it so it merges in any other values on create/update/find as opposed to just setting the ID or even leaving it as what we passed in.

Reply:
Yea, once we have serialization, we can do that.

Reply:
Kk sounds good